EnglishTranslation-master/QUEST_LV_0300.tsv

```diff
@@ -1206,30 +1206,30 @@ QUEST_LV_0300_20160310_001205	What do we do with the useless scriptures?{nl}We u
 QUEST_LV_0300_20160310_001206	Thank you for saving me!{nl}Of course, this is all because of the blessing of the Divine Tree.{nl}
 QUEST_LV_0300_20160310_001207	Why do you think the demons attacked us?{nl}That is strange... This has never happened before.{nl}
 QUEST_LV_0300_20160310_001208	Farewell then... I need to attend a ritual.{nl}Make sure you're not late either, brother. I will see you later.
-QUEST_LV_0300_20160310_001209
-QUEST_LV_0300_20160310_001210
-QUEST_LV_0300_20160310_001211
-QUEST_LV_0300_20160310_001212
-QUEST_LV_0300_20160310_001213
-QUEST_LV_0300_20160310_001214
-QUEST_LV_0300_20160310_001215
-QUEST_LV_0300_20160310_001216
-QUEST_LV_0300_20160310_001217
-QUEST_LV_0300_20160310_001218
-QUEST_LV_0300_20160310_001219
-QUEST_LV_0300_20160310_001220
-QUEST_LV_0300_20160310_001221
-QUEST_LV_0300_20160310_001222
-QUEST_LV_0300_20160310_001223
-QUEST_LV_0300_20160310_001224
-QUEST_LV_0300_20160310_001225
-QUEST_LV_0300_20160310_001226
-QUEST_LV_0300_20160310_001227
-QUEST_LV_0300_20160310_001228
-QUEST_LV_0300_20160310_001229
-QUEST_LV_0300_20160310_001230
-QUEST_LV_0300_20160310_001231
-QUEST_LV_0300_20160310_001232
+QUEST_LV_0300_20160310_001209 Villager James
```

Comment:
This applies to all the new lines you added: you should probably separate the line ID with the line itself using a tab to stay consistent with all the other lines, not a space.
example:
`QUEST_LV_0300_20160310_001209 Villager` 
to
`QUEST_LV_0300_20160310_001209	Villager`